physics/module_mp_thompson.F90

```diff
@@ -3273,7 +3291,7 @@ subroutine mp_thompson (qv1d, qc1d, qi1d, qr1d, qs1d, qg1d, ni1d,    &
             lami = cie(2)/5.E-6
             xni = MIN(4999.D3, cig(1)*oig2*xri/am_i*lami**bm_i)
             niten(k) = (xni-ni1d(k)*rho(k))*odts*orho
-           elseif (xDi.gt. 300.E-6) then
+           elseif (xDi.gt. D0s + 100.E-6) then 
```

Comment:
Same as an earlier comment.  This line should be reverted back to the original line of code.

Reply:
same as before

Reply:
Same as previously mentioned

Reply:
Reverted as suggested.